iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -164,6 +170,15 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager) {
   // Outline the dispatch regions into their own functions. This separates the
   // sequencer functions performing dispatches from the dispatchees.
   passManager.addPass(IREE::Flow::createOutlineDispatchRegionsPass());
+  passManager.addNestedPass<FuncOp>(createCanonicalizerPass());
+
+  // Export all the dispatch functions. This creates functions and feeds
+  // iree.unfoldable_constant as inputs to the exported functions.
+  if (exportDispatchFunctions) {
```

Comment:
We can separate  out this from main pipeline to run after flow pipeline:
`iree-opt --iree-flow-transformation-pipeline --iree-flow-export-dispatch-functions`

Reply:
See explanation below